frontend/config.js

```diff
@@ -6,7 +8,7 @@ module.exports = {
   ENABLE_DEV_CONTENT: (process.env.ENABLE_DEV_CONTENT === '1'),
   AVAILABLE_LOCALES: (process.env.ENABLE_DEV_LOCALES === '1') ?
     // All locales on Pontoon for local & dev
-    'en-US,ar,ast,bn-BD,bs,cs,de,dsb,es-CL,es-ES,fa,fr,fy-NL,hsb,hu,it,ja,kab,pt-BR,ru,sk,sl,sv-SE,te,tr,uk,zh-CN,zh-TW' :
+    fs.readdirSync('./locales').join(',') :
```

Comment:
Is order important? Currently "en-US" is first, but if we do `readdirSync()` it will appear in the somewhere in the first 1/3rd of the list (alphabetically).